R/user-record.R

```diff
@@ -0,0 +1,25 @@
+#' Return Crunch User Information
+#'
+#' This function returns information about the current user of the shiny app. This
+#' is useful if you want to change the behaviour of the app dependeing on who is
```

Comment:
America, please :P

Also spelling. Perhaps add the `spelling` package check while you're at it, since you're adding lots of docs?